refinery/data_set_manager/views.py

```diff
@@ -525,14 +536,15 @@ def post(self, request, *args, **kwargs):
                 custom_delimiter_string=request.POST.get(
                     'custom_delimiter_string', False
                 ),
-                identity_id=identity_id
+                identity_id=identity_id,
+                existing_dataset_uuid=existing_dataset_uuid
```

Comment:
No need for an extra variable: `existing_dataset_uuid=request.GET.get('data_set_uuid')`